objects/ConsoleExtension/Right Click Tools.json

```diff
@@ -0,0 +1,6 @@
+{
+  "downloadLocation": "https://releases.recastsoftware.com/releases/downloadcab/782",

```

Comment:
>downloadLocation [](start = 3, length = 16)

@kmyrup Last thing before we sign off on this we may want to finish your final checkin for the detonation checking and have it run on this PR. At that point I think we might be good.